src/content/self-hosted/install/releases.mdx

```diff
@@ -6,11 +6,29 @@ id: releases
 ---
 
 ## 1.11.0
-TBD
+2 August 2023
+
+### Before you upgrade
+
+Okteto 1.11 enables by default ingress-nginx validation webhook (adding an additional Kubernetes deployment), which protects the platform against single user ingress syntax errors. <!-- 6706 -->
+
+### Features
+
+- Upgrade buildkit v0.11.6. <!-- 6771 -->
```

Comment:
```suggestion
- Upgraded buildkit to v0.11.6. <!-- 6771 -->
```

This seems more like an improvement to me than a feature. Any objections to moving it to the `Improvements` section?